ui/pages/Project/components/RnaSeqResultPage.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Loader, Grid, Dropdown } from 'semantic-ui-react'
+
+import { getGenesById, getIndividualsByGuid, getRnaSeqDataByIndividual, getRnaSeqSignificantJunctionData } from 'redux/selectors'
+import { RNASEQ_JUNCTION_PADDING, TISSUE_DISPLAY } from 'shared/utils/constants'
+import DataLoader from 'shared/components/DataLoader'
+import FamilyReads from 'shared/components/panel/family/FamilyReads'
+import RnaSeqJunctionOutliersTable from 'shared/components/table/RnaSeqJunctionOutliersTable'
+import { loadRnaSeqData } from '../reducers'
+import { getRnaSeqDataLoading } from '../selectors'
+
+const RnaSeqOutliers = React.lazy(() => import('./RnaSeqOutliers'))
+
+const OUTLIER_VOLCANO_PLOT_CONFIGS = [
+  {
+    key: 'outliers',
+    getLocation: (({ geneId }) => geneId),
+    searchType: 'genes',
+    title: 'Expression Outliers',
+    formatData: data => data,
+  },
+  {
+    key: 'spliceOutliers',
+    getLocation: (({ chrom, start, end }) => `${chrom}:${Math.max(1, start - RNASEQ_JUNCTION_PADDING)}-${end + RNASEQ_JUNCTION_PADDING}`),
+    searchType: 'regions',
+    title: 'Splice Junction Outliers',
+    formatData: (data, tissueType) => data.filter(outlier => outlier.tissueType === tissueType),
+  },
+]
+
+const OutliersTableWithReads = props => <FamilyReads layout={RnaSeqJunctionOutliersTable} noTriggerButton {...props} />
+
+class BaseRnaSeqResultPage extends React.PureComponent {
+
+  static propTypes = {
+    individual: PropTypes.object,
+    rnaSeqData: PropTypes.object,
+    significantJunctionOutliers: PropTypes.arrayOf(PropTypes.object),
+    genesById: PropTypes.object,
+  }
+
+  constructor(props) {
+    super(props)
+    const { rnaSeqData } = props
+    const tissueTypes = Array.from(Object.values(rnaSeqData?.spliceOutliers || {}).flat().reduce(
+      (acc, { tissueType }) => acc.add(tissueType), new Set(),
+    ))
+    const tissueOptions = tissueTypes.map(tissueType => (
```

Comment:
as stated in previous PR review, `tissueOptions` do not belong in the state as they can not change, and need to be created using `createSelector` and assigned as a prop